src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -81,6 +79,18 @@ private function recursiveClean($data): array
         return $data;
     }
 
+    /**
+     * @param \ArrayObject|array $value
+     */
+    private function ksort(&$value)
+    {
+        if ($value instanceof \ArrayObject) {
+            $value->ksort();
+        } elseif (\is_array($value)) {
```

Comment:
Should you use an `elseif`? If it's not an `ArrayObject`, it's an array, according to the PHPDoc.